lib/stack_master/identity.rb

```diff
@@ -0,0 +1,23 @@
+module StackMaster
+  class Identity
+    def running_in_allowed_account?(allowed_accounts)
```

Comment:
Seems odd to pass a `nil` or `empty` variable to a function.

Reply:
Maybe if `nil` was the default value for allowed_accounts (in StackDefinition, indicating unspecified) the expected values would either be `nil` or an array of account IDs

Reply:
Good idea. Done in b30dede.